src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,36 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    categories: {
+      id: null,
+      name: '',
+    },
+    categoryList: [
+      {
+        id: 1,
+        name: 'aaa',
+      },
+    ],
```

Comment:
[imo]

配列の初期値を設定したい場合、空配列で定義すると良いかなと思います。

```suggestion
    categoryList: [],
```

Reply:
空の配列に定義し直しました。ありがとうございます